### PR TITLE
feat: TEC-1147 add purchaseRights icon stub

### PR DESCRIPTION
**Pull request by ricardofbarros**
NOTE: This is just a icon stub. When the real one is finished we just push a new PR with the new asset.

### CODE REVIEW

File: icons/purchase-rights.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="1024px" height="1024px" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.5.2 (25235) - http://www.bohemiancoding.com/sketch -->
```
Comment (umbobabo) [inline icons/purchase-rights.svg]: @ricardofbarros Unfortunately the code generated by Sketch is not ready to be used as is.
Please compare this svg to another icon and you should see that useless tags and attributes have been stripped out manually.
For example if there is a <desc> should not be like the one 'Created by Sketch'.

Comment (fabiosantoscode) [inline icons/purchase-rights.svg]: These are stripped by our icon CSS builder and by our icon aggregator if I recall correctly

Comment (fabiosantoscode) [inline icons/purchase-rights.svg]: But one thing I can see is that the floats are WAY too precise here. Lots of decimal places. We don't need any.

**Comment by ricardofbarros** [comment]
Hey @umbobabo I updated the icon with the econ svg, is that alright?

File: icons/purchase-rights.svg (hunk 1 above)
Comment (ricardofbarros) [inline icons/purchase-rights.svg]: I copied from print.svg, but if you see all svg icons most of them have these fields @umbobabo @fabiosantoscode

**Comment by fabiosantoscode** [comment]
LGTM

**Comment by ricardofbarros** [comment]
@umbobabo can I merge it?

**Pull request closed without merging**